src/main/resources/application.yml

```
@@ -0,0 +1,4 @@
+# profile name = dev, prod
+spring:
+  profiles:
+    active: oauth, dev
```

```suggestion
spring:
  profiles:
    group:
      "test": "test"
      "local": "local, datasource, oauth"
      "dev": "dev, datasource, oauth"
      "prod": 생략
```

보통 local에서 개발을 하고 dev(테스트서버)에 배포하여 테스트하고 prod(상용)환경에 배포가 된다고 하네용

datasource도 나누어지면 좋을 것 같아요

---

몰랐던 사실이네요
datasource도 나뉘게 된다면 서버 별로 별도로 구성해야 하는걸까요??

---

하나의 datasource에서 환경변수에 어떤 값을 넣냐에 따라 환경이 달라지도록 설정하면 좋을 것 같습니다.